source/documentation/administration_guide/topics/Understanding_Storage.adoc

```diff
@@ -15,7 +15,7 @@ endif::rhv-doc[]
 
 On NFS, all virtual disks, templates, and snapshots are files.
 
-On SAN (iSCSI/FCP), each virtual disk, template or snapshot is a logical volume. Block devices are aggregated into a logical entity called a volume group, and then divided by LVM (Logical Volume Manager) into logical volumes for use as virtual hard disks. See _{enterprise-linux} Logical Volume Manager Administration Guide_ for more information on LVM.
+On SAN (iSCSI/FCP), each virtual disk, template or snapshot is a logical volume. Block devices are aggregated into a logical entity called a volume group, and then divided by LVM (Logical Volume Manager) into logical volumes for use as virtual hard disks. See _{enterprise-linux} Configuring and managing logical volumes_  guide for more information on LVM.
```

Comment:
First sentence: Add a serial comma after "template".
Second sentence: Remove comma after "volume group".
Third sentence: Replace attribute with RHEL product name. 
Remove extra space after title. 
Remove  "guide" and "on LVM".